### DISCUSSION TITLE
Release v1.9.1 / 5.64.1

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2850] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This builds fixes a memory corruption issue with privacy enhanced sandboxes, as well as fixes other bugs
last but not least it improves the UI and adds some additional protection features.

Please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#191--5641---2023-04-23) for more details.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.1'>Release v1.9.1 / 5.64.1</a>.</em>

**Comment by BilleBarrett** [comment]
update a clients computer and like mine Microsoft Edge comes up with a Black background 
Google Chrome seems to be OK
Is there a setting that needs to be dene
